build.sbt

```diff
@@ -1507,7 +1507,8 @@ lazy val `distage-framework` = project.in(file("distage/distage-framework"))
       "org.typelevel" %% "cats-effect" % V.cats_effect % Optional,
       "dev.zio" %% "zio" % V.zio % Optional,
       "io.monix" %% "monix-bio" % V.monix_bio % Optional,
-      "org.scala-lang" % "scala-reflect" % scalaVersion.value % Provided
+      "org.scala-lang" % "scala-reflect" % scalaVersion.value,
```

Comment:
@VladPodilnyk Could please you return `Provided` here and use Optional in interop-cats, not Provided?